sky/clouds/service_catalog/__init__.py

```diff
@@ -18,11 +18,12 @@
 _ALL_CLOUDS = ('aws', 'azure', 'gcp')
 
 
-def _map_clouds_catalog(clouds: CloudFilter, method_name, *args, **kwargs):
+def _map_clouds_catalog(clouds: CloudFilter, method_name: str, *args, **kwargs):
     if clouds is None:
         clouds = list(_ALL_CLOUDS)
     single = isinstance(clouds, str)
     if single:
+        assert isinstance(clouds, str)
```

Comment:
Debugging remnant? Given above two lines, isinstance(clouds, str) should always be True

Reply:
It is a "bug" of mypy, which cannot propagate the value of `isinstance` through boolean variable ([reference](https://github.com/python/mypy/issues/6670))

I switched to `# type: ignore` annotation to get rid of the complain from mypy. Does that look better?